query/rds/rds_snapshot_private_access.sql

```diff
@@ -0,0 +1,39 @@
+(
```

Comment:
What does "private access" mean here? What is this query really checking for?

Reply:
@cbruno10 `private access` here generally  means that the snapshots should not be publicly accessible for restoration etc. We already have this query `rds_snapshot_prohibit_public_access.sql`(which is same as `rds_snapshot_private_access.sql` query) but for the reason change we have added a new query file as the title in FSBP is `[RDS.1] RDS snapshots should be private`.
https://turbothq.slack.com/archives/C01RN2G6KNK/p1622200538038400

Reply:
We should just use `rds_snapshot_prohibit_public_access.sql` for this, as the control titles per framework will be different anyway, so we should rely on good base query naming/reuse.